ctrl.c

```diff
@@ -1318,8 +1362,23 @@ void pv_ctrl_free_cmd(struct pv_cmd *cmd)
 static int pv_ctrl_init(struct pv_init *this)
 {
 	struct pantavisor *pv = pv_get_instance();
+	struct statfs fs;
 	char path[PATH_MAX];
 
+	if (statfs("/sys/fs/cgroup/", &fs) < 0) {
+		pv_log(ERROR, "statfs /sys/fs/cgroup/ failed: %s",
+		       strerror(errno));
+		return -1;
+	}
+
+	if (fs.f_type == CGROUP2_SUPER_MAGIC) {
+		pv_log(INFO, "system is using cgroup v2");
+		pv->cgroupv = 2;
```

Comment:
we should here really also determine mixed mode.... mixed is if cgroup2 is mounted on /sys/fs/cgroup//unified.... mixed will allow us later to better filter/generate the cgroup2 / cgroup1 rules i guess

Reply:
Added this. Now we detect if in v2 with as with the previous version. If not in v2, we check if the unified file exists or not to determine whether we are in unified or v1.

Reply:
done